bazaarvoice/emodb
# TASK:193026 | Critical Vulnerabilties fixed

TASK:193026 | Critical Vulnerabilties fixed

## Github Issue #

[Story Link](https://bazaarvoice.atlassian.net/browse/PD-193026)

## What Are We Doing Here?

Currently there were critical many vulnerabilities found , the changes made as part of this story will fix these issues.

## How to Test and Verify

  Tested locally.
![16B0FD00-4A22-4A15-81AA-5798439D9F85](https://github.com/bazaarvoice/emodb/assets/152610061/7309ee41-45e4-4798-ba81-5ece5e78cf16)

1. Check out this PR
2. Run Command X, Click Button Y
3. Profit

## Risk

### Level 

Low

### Required Testing

`Smoke`, `Regression`, or `Manual`. (All changes except documentation need smoke
testing at a minimum).

### Risk Summary

Add one or a few complete sentences about the possible risks or concerns for
this change.

## Code Review Checklist


build and run.

we inevitably come back to this code it will only take hours to figure out, not
days.

a victim of rampaging consistency, and should be using this course of action. 
We don't have coding standards out yet for this project, so please make sure to address any feedback regarding STYLE so the codebase remains consistent.
